tests/lib/hardware/openstack_libcloud.py

```diff
@@ -305,6 +309,25 @@ def _get_ex_network_by_name(self, name=None):
 
         return None
 
+    def _create_network_private(self):
+        """
+        create the openstack network/subnet/router and set external gateway
+        """
+        logger.info("creating openstack network/subnet/router")
+        network_public = self._get_ex_network_by_name(
+            name=settings.OS_EXTERNAL_NETWORK)
+        network = self.conn.ex_create_network(f"{self.workspace.name}-net")
+        subnet = self.conn.ex_create_subnet(f"{self.workspace.name}-subnet",
+                                            network, "192.168.100.0/24")
```

Comment:
Is this subnet generally safe for any openstack cloud? Otherwise we should maybe make it configurable (or perhaps regardless).

Reply:
don't know if rookcheck is gonna be used to test scaling deployments, but /16 might be a better subnet